### ISSUE TITLE
Development Road Map

### ISSUE THREAD

**Issue by devmsh** [unitoneict/baas#1]
To achieve the best implementation for the Baas

* It will developed based on Laravel Framework
* The specification will be analyzed and cloned from https://parse.com/docs/rest/guide
* We will use Parse PHP SDK and it test suite to make sure that our implementation successfully replace the Parse hosted service
* Once we pass the PHP SDK test suite, we will work to pass the Android/iOS SDK test suite also.

Currently we will work to support:

* [ ] Objects endpoints
* [ ] Queries endpoints
* [ ] Users endpoints

Then we will continue with the others endpoints

**Comment by yeahio-com** [comment]
Are you still working on this project?

**Comment by devmsh** [comment]
Yes, we use it internally, and we will make another push in the next few days
